buildSrc/src/main/groovy/org/elasticsearch/gradle/doc/SnippetsTask.groovy

```diff
@@ -30,13 +30,13 @@ class SnippetsTask extends DefaultTask {
     private static final String SCHAR = /(?:\\\/|[^\/])/
     private static final String SUBSTITUTION = /s\/($SCHAR+)\/($SCHAR*)\//
     private static final String CATCH = /catch:\s*((?:\/[^\/]+\/)|[^ \]]+)/
-    private static final String SKIP = /skip:([^\]]+)/
+    private static final String SKIP_REGEX = /skip:([^\]]+)/
```

Comment:
$SKIP seems like a reserved term in Groovy 3.0 and caused trouble here. Therefore the renaming